nginx/default.conf

```diff
@@ -39,9 +39,10 @@ server {
     send_timeout 1500;
   }
 
-  location /whyis {
-    rewrite /whyis/(.*) /$1 break;
-    proxy_pass http://whyis;
+  # location /whyis {
+  #   rewrite /whyis/(.*) /$1 break;
+  #   proxy_pass http://whyis;
+  # }
```

Comment:
Are these changes intended for this ticket?

Reply:
Similar to the other one -- I added the closing curly brace in line 45, but have commented for now since it seems like there might also be other things missing addressed in PR #169